README.md

```diff
@@ -70,50 +82,70 @@ The default template uses JUnit 5. You may be used to JUnit 4, but here are some
 
 1. Perform an `oc login` to your Open Shift cluster for development purposes.
 
-1.  Get the [odo v.1.2.6](https://mirror.openshift.com/pub/openshift-v4/clients/odo/v1.2.6/) CLI (or later).
+1.  Install the latest vesion of OpenShift Do (odo) using [these instructions](https://odo.dev/docs/installing-odo/)
 
-1. Odo must be run in experimental mode at the 1.x level in order to access the devfile support. Enable experimental mode using the odo CLI:
+    <!-- Keep for now in case future changes require experimental mode
+    1. Odo must be run in experimental mode at the 1.x level in order to access the devfile support. Enable experimental mode using the odo CLI:
     ```shell
     odo preference set experimental true
     ```
+    --->
 
 1. Create a new folder in your local directory and create an odo project using the Odo CLI, e.g.:
     ```shell
     mkdir my-project
     cd my-project
-    odo project create <project namespace name> 
+    odo project create <project namespace name>
     ```
     The project namespace name will be used to create an openshift namespace that will house your application and related pods
 
-1. Initialize the local folder for the odo project
+1. Initialize the local folder using the Open Liberty stack.
     ```shell
-    odo create java-openliberty <app project name> --starter
+    odo create java-openliberty --starter
     ```
-    
+
     This will initialize an Open Liberty project and download the default starter app artifacts.
 
-1. Create an odo URL that will be used to access the starter app:
-    ```shell
-    odo url create --port 9080
-    ```
-1. Once your project has been initialized and the URL created, you can push your application to your openshift cluster for the first time using the following command:
+1. Once your project has been initialized, you can push your application to your openshift cluster for the first time using the following command:
```

Comment:
I think a separate section or mention of this working with minikube is enough. Since the main path is using OpenShift, it would be good to keep the default doc with just OpenShift. If we see it becoming a more common use case, then a separate article might be good.